### ISSUE TITLE
Sometimes survey retrieval appears to be successful when in fact it isn't

### ISSUE THREAD

**Issue by efcaguab** [WorldFishCenter/peskas.timor.data.pipeline#9]
As an example see the logs from https://github.com/WorldFishCenter/peskas.timor.data.pipeline/pull/8/checks?check_run_id=2190423335:

```
Run Rscript -e 'peskas.timor.data.pipeline::ingest_timor_landings()'
INFO [2021-03-25 05:02:33] Loading configuration file...
INFO [2021-03-25 05:02:33] Using configutation: default
DEBUG [2021-03-25 05:02:33] Running with parameters list(landings = list(api = "kobohr", survey_id = 344563, token = "***", file_prefix = "timor-landings-v2"), landings_legacy = list(api = "kobohr", survey_id = NULL, token = "***", file_prefix = "timor-landings-v1"))
DEBUG [2021-03-25 05:02:33] Running with parameters list(google = list(options = list(project = "peskas", bucket = "timor-dev", service_account_key = "***"), key = "gcs"))
INFO [2021-03-25 05:02:33] Downloading survey metadata as timor-landings-v2_metadata__20210325050233_34392a7__.json...
SUCCESS [2021-03-25 05:02:34] Metadata download succeeded
INFO [2021-03-25 05:02:34] Downloading survey csv data as timor-landings-v2_raw__20210325050233_34392a7__.csv...
SUCCESS [2021-03-25 05:04:35] Survey csv data download succeeded
INFO [2021-03-25 05:04:35] Downloading survey json data as timor-landings-v2_raw__20210325050233_34392a7__.json...
SUCCESS [2021-03-25 05:06:36] Survey json data download succeeded
INFO [2021-03-25 05:06:36] Uploading files to cloud...
2021-03-25 05:06:36 -- File size detected as 72.3 Kb
2021-03-25 05:06:37 -- File size detected as 157 bytes
2021-03-25 05:06:37 -- File size detected as 157 bytes
SUCCESS [2021-03-25 05:06:38] File upload succeded
```

The file sizes are extremely small, so probably the API requested failed and the content message (probably an error request status) is being saved as the data or metadata.

Fixing this bug probably requires:

- [ ] Inspecting the response content and check the format corresponds with expectations
- [ ] Inspecting the response code and logging it so that it's easier to understand the source of the errors

**Comment by efcaguab** [comment]
Another example of this situation: https://github.com/WorldFishCenter/peskas.timor.data.pipeline/runs/2443790857?check_suite_focus=true

**Comment by efcaguab** [comment]
Fixed in #39
